src/user/user.spec.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-env jest */
+
+import {
+  xo, testConnection,
+} from "../util";
+import { createUser, deleteUsers } from "./helper";
+
+describe("user", () => {
+  let userIds = [];
+
+  afterAll(async () => {
+    await deleteUsers(xo, userIds);
+    userIds = [];
+  });
+
+  describe(".create()", () => {
+
+    describe("successful tests", () => {
+      let email;
+      let password;
+
+      afterEach(async () => {
+        await testConnection({ email, password })
+      });
```

Comment:
In this way, I have to manage "email" and "password" in two variables. Is it simpler to test the connection as is done now?